aws-api-request.js

```diff
@@ -138,7 +137,7 @@ function awsApiRequest(options, retryAttempt = 0) {
                     //2~8 * 100 = 25600
                     //2~9 * 100 = 51200
 
-                    if (retryAttempt > MAX_RETRY_COUNT) {
```

Comment:
This sort of makes the `MAX_RETRY_COUNT` variable useless. 

I think you can just delete it.